src/api/services/enrichment/enrichment.js

```diff
@@ -31,7 +37,10 @@ export const createEnrichmentRule = async ctx => {
 const isDirectPath = sourceData => {
     return (
         typeof sourceData === 'string' ||
-        (Array.isArray(sourceData) && typeof sourceData[0] === 'string')
+        typeof sourceData === 'number' ||
```

Comment:
Can we make this more readable with functions and/or intermediate well named variables?